src/cpu/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
-#===============================================================================
-# Copyright 2019-2021 Intel Corporation
-# Copyright 2020 Arm Ltd. and affiliates
-# Copyright 2021 FUJITSU LIMITED
+#== == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == =
+#Copyright 2019 - 2021 Intel Corporation
+#Copyright 2020 Arm Ltd.and affiliates
+#Copyright 2021 FUJITSU LIMITED
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#Licensed under the Apache License, Version 2.0(the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
+#http: //www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#===============================================================================
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#== == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == == =
```

Comment:
Could you drop the modification of copyright, please?

Reply:
Done